### DISCUSSION TITLE
No stats

### DISCUSSION THREAD

**Discussion by zenitsuy** [ppy/osu#17512] [Q&A] [answered]
### Type

Game behaviour

### Bug description

So I started playing osu! on Linux so I use Lazer cause stable doesn't work with wine very well (at least for me) . I don't know why but I'm playing lots of hours and my playtime is visible on my profile. But my stats do not show up anywhere.
Is this some kind of bug or am I just dumb?

### Version

2022.327.0

### Logs

[database.log](https://github.com/ppy/osu/files/8363806/database.log)
[network.log](https://github.com/ppy/osu/files/8363807/network.log)
[performance-draw.log](https://github.com/ppy/osu/files/8363808/performance-draw.log)
[performance-update.log](https://github.com/ppy/osu/files/8363809/performance-update.log)
[performance.log](https://github.com/ppy/osu/files/8363810/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8363811/runtime.log)

**Comment by peppy** [comment, marked as answer]
which stats. which user.

**Reply by zenitsuy** [reply]
Username: Inarey

**Reply by zenitsuy** [reply]
It just says i had no recent plays

**Reply by peppy** [reply]
![Safari 2022-03-28 at 14 50 12](https://user-images.githubusercontent.com/191335/160425135-9f4e0e8c-ef95-4e9e-b29a-445ddae039f5.png)

**Reply by zenitsuy** [reply]
Sorry. :/ Didn't show up on my page sorry. Thanks for the quick answer :)
